samples/DotNet/GettingStarted/BasicSendReceiveQuickStart/BasicSendReceiveQuickStart/Program.cs

```diff
@@ -8,41 +8,41 @@ namespace BasicSendReceiveQuickStart
 {
     class Program
     {
-        static IQueueClient queueClient;
+        static IQueueClient _queueClient;
 
         static void Main(string[] args)
         {
-            string ServiceBusConnectionString = "";
-            string QueueName = "";
+            string serviceBusConnectionString = string.Empty;
+            string queueName = string.Empty;
 
-            for (int i = 0; i < args.Length; i++)
+            for (var i = 0; i < args.Length; i++)
```

Comment:
I wonder why a `var` here, but not two lines above?

Reply:
Good point, changed.